manifests/params.pp

```diff
@@ -229,6 +229,10 @@
   $templates_listen_on = 'both'
   $template_url        = "http://${facts['networking']['fqdn']}:${http_port}"
 
+  # Registration settings
+  $registration           = true
+  $registration_listen_on = 'both'
```

Comment:
Do we need this to be available on HTTP? Aren't we trying to get HTTPS working to make it secure?